Foundation/Process.swift

```diff
@@ -271,31 +271,31 @@ open class Task: NSObject {
             
             repeat {
 #if CYGWIN
-                waitResult = waitpid( task.processIdentifier, exitCodePtrWrapper, 0)
+                waitResult = waitpid( process.processIdentifier, exitCodePtrWrapper, 0)
 #else
-                waitResult = waitpid( task.processIdentifier, &exitCode, 0)
+                waitResult = waitpid( process.processIdentifier, &exitCode, 0)
 #endif
             } while ( (waitResult == -1) && (errno == EINTR) )
             
-            task.terminationStatus = WEXITSTATUS( exitCode )
+            process.terminationStatus = WEXITSTATUS( exitCode )
             
             // If a termination handler has been set, invoke it on a background thread
             
-            if task.terminationHandler != nil {
+            if process.terminationHandler != nil {
                 let thread = Thread {
-                    task.terminationHandler!(task)
+                    process.terminationHandler!(process)
                 }
                 thread.start()
             }
             
             // Set the running flag to false
             
-            task.running = false
+            process.running = false
```

Comment:
The property should be renamed to `isRunning` too: https://developer.apple.com/reference/foundation/process/1415788-isrunning